orders-mongo/src/main/java/io/helidon/examples/sockshop/orders/mongo/MongoOrderRepository.java

```diff
@@ -23,11 +25,11 @@
  * that that uses MongoDB as a backend data store.
  */
 @ApplicationScoped
-@Specializes
+@Alternative
+@Priority(Interceptor.Priority.APPLICATION+10)
```

Comment:
Make priority `APPLICATION` -- higher than default and default test implementation.